### PR TITLE
added fix to display calendar in load demo form

### PR DESCRIPTION
**Pull request by AmitPhulera**
https://dimagi-dev.atlassian.net/browse/SAAS-10773
##### SUMMARY

Fixes the issue because of which calendar was not loading in load demo form.

### CODE REVIEW

File: corehq/apps/analytics/static/analytix/js/hubspot.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ hqDefine('analytix/js/hubspot', [
                 });
             },
             onFormSubmit: function ($form) {
-                $('#get-demo-cta-calendar-content').toggleClass("hidden");
+                $('#get-demo-cta-calendar-content').fadeIn();
```
Comment (orangejenny) [inline corehq/apps/analytics/static/analytix/js/hubspot.js:94]: Why is this inconsistent with the line below? This seems easy for someone to innocently re-break in the future.

Comment (AmitPhulera) [reply corehq/apps/analytics/static/analytix/js/hubspot.js:94]: Both the forms(Request Trial and Request Demo) have the same functions exposed so accidentally "Request Demo" form was edited while I changed the "Trail Form", I just reverted it how it was before.

Should I make changes to the demo form also, which I implemented in the trial form?

Comment (orangejenny) [reply corehq/apps/analytics/static/analytix/js/hubspot.js:94]: Yeah, I think so. But no need to do it in this PR if you want to get the bugfix out.

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
Test failure is unrelated. It's fine not to run tests on this PR - we do have js tests, but they don't cover the analytics code.

**Pull request merged**